README.md

```diff
@@ -26,12 +26,30 @@ $ sh build.sh
 
 ## Demo
 ### gauss_pdf_and_cdf.py
```

Comment:
Do you think the name of the file is a good title to show users what they will see?